diff --git a/node_modules/hexo-theme-next/layout/_partials/footer.njk b/node_modules/hexo-theme-next/layout/_partials/footer.njk
index cffc669..52a6277 100644
--- a/node_modules/hexo-theme-next/layout/_partials/footer.njk
+++ b/node_modules/hexo-theme-next/layout/_partials/footer.njk
@@ -13,7 +13,7 @@
 {%- if theme.footer.copyright !== false %}
   <div class="copyright">
     {%- set copyright_year = date(null, 'YYYY') %}
-    &copy; {% if theme.footer.since and theme.footer.since != copyright_year %}{{ theme.footer.since }} – {% endif %}
+    <span class="copyright-to-rotate">&copy;</span> {% if theme.footer.since and theme.footer.since != copyright_year %}{{ theme.footer.since }} – {% endif %}
     <span itemprop="copyrightYear">{{ copyright_year }}</span>
     <span class="with-love">
       <i class="{{ theme.footer.icon.name }}"></i>
diff --git a/node_modules/hexo-theme-next/layout/_partials/widgets.njk b/node_modules/hexo-theme-next/layout/_partials/widgets.njk
index b312d0a..bf5d292 100644
--- a/node_modules/hexo-theme-next/layout/_partials/widgets.njk
+++ b/node_modules/hexo-theme-next/layout/_partials/widgets.njk
@@ -13,6 +13,7 @@
   <div class="back-to-top" role="button" aria-label="{{ __('accessibility.back_to_top') }}">
     <i class="fa fa-arrow-up fa-lg"></i>
     <span>0%</span>
+    <span><code>M-a</code></span>
   </div>
 {%- endif %}
 
diff --git a/node_modules/hexo-theme-next/source/js/sidebar.js b/node_modules/hexo-theme-next/source/js/sidebar.js
index 6ab3c6b..ba5b3dd 100644
--- a/node_modules/hexo-theme-next/source/js/sidebar.js
+++ b/node_modules/hexo-theme-next/source/js/sidebar.js
@@ -11,6 +11,12 @@ document.addEventListener('DOMContentLoaded', () => {
       window.addEventListener('mouseup', this.mouseupHandler.bind(this));
       document.querySelector('.sidebar-dimmer').addEventListener('click', this.clickHandler.bind(this));
       document.querySelector('.sidebar-toggle').addEventListener('click', this.clickHandler.bind(this));
+      document.addEventListener('keydown', (event) => {
+        if (event.ctrlKey && event.key === 'd') {
+          event.preventDefault();
+          this.clickHandler.bind(this)();
+        }
+      });
       window.addEventListener('sidebar:show', this.showSidebar);
       window.addEventListener('sidebar:hide', this.hideSidebar);
     },
diff --git a/node_modules/hexo-theme-next/source/js/third-party/search/local-search.js b/node_modules/hexo-theme-next/source/js/third-party/search/local-search.js
index 92a264d..a04f649 100644
--- a/node_modules/hexo-theme-next/source/js/third-party/search/local-search.js
+++ b/node_modules/hexo-theme-next/source/js/third-party/search/local-search.js
@@ -67,13 +67,20 @@ document.addEventListener('DOMContentLoaded', () => {
   window.addEventListener('search:loaded', inputEventFunction);
 
   // Handle and trigger popup window
+  function searchPopupClick() {
+    document.body.classList.add('search-active');
+    // Wait for search-popup animation to complete
+    setTimeout(() => input.focus(), 500);
+    if (!localSearch.isfetched) localSearch.fetchData();
+  }
   document.querySelectorAll('.popup-trigger').forEach(element => {
-    element.addEventListener('click', () => {
-      document.body.classList.add('search-active');
-      // Wait for search-popup animation to complete
-      setTimeout(() => input.focus(), 500);
-      if (!localSearch.isfetched) localSearch.fetchData();
-    });
+    element.addEventListener('click', searchPopupClick);
+  });
+  document.addEventListener('keydown', (event) => {
+    if (!document.body.classList.contains('search-active') && event.ctrlKey && event.key === 's') {
+      event.preventDefault();
+      searchPopupClick();
+    }
   });
 
   // Monitor main search box
diff --git a/node_modules/hexo-theme-next/source/js/utils.js b/node_modules/hexo-theme-next/source/js/utils.js
index ebd8552..22d9f13 100644
--- a/node_modules/hexo-theme-next/source/js/utils.js
+++ b/node_modules/hexo-theme-next/source/js/utils.js
@@ -199,6 +199,17 @@ NexT.utils = {
         scrollTop: 0
       });
     });
+    document.addEventListener('keydown', (event) => { 
+      if (event.altKey && event.key === 'a') {
+        event.preventDefault();
+        window.anime({
+          targets  : document.scrollingElement,
+          duration : 200,
+          easing   : 'linear',
+          scrollTop: 0
+        });
+      }
+    });
   },
 
   /**
