diff --git a/node_modules/hexo-theme-next/layout/_macro/post.njk b/node_modules/hexo-theme-next/layout/_macro/post.njk
index 304726c..1460c35 100644
--- a/node_modules/hexo-theme-next/layout/_macro/post.njk
+++ b/node_modules/hexo-theme-next/layout/_macro/post.njk
@@ -49,6 +49,12 @@
           {%- endif %}
         </{% if is_index %}h2{% else %}h1{% endif %}>
 
+        {%- if post.subtitle and not is_index %}
+          <h1 class="post-subtitle">
+            〜{{ post.subtitle }}〜
+          </h1>
+        {%- endif %}
+
         <div class="post-meta-container">
           {{ partial('_partials/post/post-meta.njk') }}
 
diff --git a/node_modules/hexo-theme-next/layout/_partials/widgets.njk b/node_modules/hexo-theme-next/layout/_partials/widgets.njk
index b312d0a..b9e3efa 100644
--- a/node_modules/hexo-theme-next/layout/_partials/widgets.njk
+++ b/node_modules/hexo-theme-next/layout/_partials/widgets.njk
@@ -7,12 +7,16 @@
     <span class="toggle-line"></span>
   </div>
   <div class="sidebar-dimmer"></div>
+  <div class="sidebar-hotkey-info">
+    <code>CTRL+D</code>
+  </div>
 {%- endif %}
 
 {%- if theme.back2top.enable and not theme.back2top.sidebar %}
   <div class="back-to-top" role="button" aria-label="{{ __('accessibility.back_to_top') }}">
     <i class="fa fa-arrow-up fa-lg"></i>
     <span>0%</span>
+    <span><code>CTRL+A</code></span>
   </div>
 {%- endif %}
 
diff --git a/node_modules/hexo-theme-next/source/css/_common/components/post/post-header.styl b/node_modules/hexo-theme-next/source/css/_common/components/post/post-header.styl
index a5f9184..7a4d0fb 100644
--- a/node_modules/hexo-theme-next/source/css/_common/components/post/post-header.styl
+++ b/node_modules/hexo-theme-next/source/css/_common/components/post/post-header.styl
@@ -30,6 +30,32 @@
   }
 }
 
+.posts-expand .post-subtitle {
+  font-size: $font-size-large;
+  font-weight: normal;
+  margin: initial;
+  word-wrap();
+
+  if (hexo-config('post_edit.enable')) {
+    .post-edit-link {
+      border-bottom: 0;
+      color: $grey;
+      float: right;
+      font-size: $font-size-small;
+      margin-left: -1.2em;
+      transition: color $transition-ease-in;
+
+      +mobile-small() {
+        margin-left: initial;
+      }
+
+      &:hover {
+        color: $sidebar-highlight;
+      }
+    }
+  }
+}
+
 .posts-expand .post-title-link {
   border-bottom: 0;
   color: var(--link-color);
diff --git a/node_modules/hexo-theme-next/source/css/_common/outline/sidebar/sidebar-toggle.styl b/node_modules/hexo-theme-next/source/css/_common/outline/sidebar/sidebar-toggle.styl
index 2b5273d..a2b5eee 100644
--- a/node_modules/hexo-theme-next/source/css/_common/outline/sidebar/sidebar-toggle.styl
+++ b/node_modules/hexo-theme-next/source/css/_common/outline/sidebar/sidebar-toggle.styl
@@ -19,3 +19,16 @@
 .sidebar-active .sidebar-toggle {
   toggle-close($sidebar-toggle-alignment);
 }
+
+.sidebar-hotkey-info {
+  bottom: 61px;
+  height: 16px;
+  padding: 5px;
+  background: transparent;
+  position: fixed;
+  z-index: 30;
+  left: 60px;
+  display: inline-flex;
+  align-items: center;
+}
+
diff --git a/node_modules/hexo-theme-next/source/js/sidebar.js b/node_modules/hexo-theme-next/source/js/sidebar.js
index 3c49861..535b0eb 100644
--- a/node_modules/hexo-theme-next/source/js/sidebar.js
+++ b/node_modules/hexo-theme-next/source/js/sidebar.js
@@ -11,6 +11,12 @@ document.addEventListener('DOMContentLoaded', () => {
       window.addEventListener('mouseup', this.mouseupHandler.bind(this));
       document.querySelector('.sidebar-dimmer').addEventListener('click', this.clickHandler.bind(this));
       document.querySelector('.sidebar-toggle').addEventListener('click', this.clickHandler.bind(this));
+      document.addEventListener('keydown', (event) => {
+        if (event.ctrlKey && event.key === 'd') {
+          event.preventDefault();
+          this.clickHandler.bind(this)();
+        }
+      });
       window.addEventListener('sidebar:show', this.showSidebar);
       window.addEventListener('sidebar:hide', this.hideSidebar);
     },
diff --git a/node_modules/hexo-theme-next/source/js/third-party/search/local-search.js b/node_modules/hexo-theme-next/source/js/third-party/search/local-search.js
index d08e119..37530e9 100644
--- a/node_modules/hexo-theme-next/source/js/third-party/search/local-search.js
+++ b/node_modules/hexo-theme-next/source/js/third-party/search/local-search.js
@@ -7,9 +7,9 @@ document.addEventListener('DOMContentLoaded', () => {
     return;
   }
   const localSearch = new LocalSearch({
-    path             : CONFIG.path,
+    path: CONFIG.path,
     top_n_per_article: CONFIG.localsearch.top_n_per_article,
-    unescape         : CONFIG.localsearch.unescape
+    unescape: CONFIG.localsearch.unescape
   });
 
   const input = document.querySelector('.search-input');
@@ -55,14 +55,20 @@ document.addEventListener('DOMContentLoaded', () => {
   window.addEventListener('search:loaded', inputEventFunction);
 
   // Handle and trigger popup window
+  function searchPopupClick() {
+    document.body.classList.add('search-active');
+    // Wait for search-popup animation to complete
+    setTimeout(() => input.focus(), 500);
+    if (!localSearch.isfetched) localSearch.fetchData();
+  }
   document.querySelectorAll('.popup-trigger').forEach(element => {
-    element.addEventListener('click', () => {
-      NexT.utils.setGutter();
-      document.body.classList.add('search-active');
-      // Wait for search-popup animation to complete
-      setTimeout(() => input.focus(), 500);
-      if (!localSearch.isfetched) localSearch.fetchData();
-    });
+    element.addEventListener('click', searchPopupClick);
+  });
+  document.addEventListener('keydown', (event) => {
+    if (!document.body.classList.contains('search-active') && event.ctrlKey && event.key === 's') {
+      event.preventDefault();
+      searchPopupClick();
+    }
   });
 
   // Monitor main search box
diff --git a/node_modules/hexo-theme-next/source/js/utils.js b/node_modules/hexo-theme-next/source/js/utils.js
index 966a320..39ffb54 100644
--- a/node_modules/hexo-theme-next/source/js/utils.js
+++ b/node_modules/hexo-theme-next/source/js/utils.js
@@ -213,6 +213,17 @@ NexT.utils = {
         scrollTop: 0
       });
     });
+    document.addEventListener('keydown', (event) => { 
+      if (event.ctrlKey && event.key === 'a') {
+        event.preventDefault();
+        window.anime({
+          targets  : document.scrollingElement,
+          duration : 200,
+          easing   : 'linear',
+          scrollTop: 0
+        });
+      }
+    });
   },
 
   /**
